pkg/operator/edgefs/cluster/mgr/mgr.go

```diff
@@ -577,6 +582,21 @@ func (c *Cluster) mgrContainer(name string, containerImage string) v1.Container
 	return cont
 }
 
+func (c *Cluster) getLivenessProbe() *v1.Probe {
+	return &v1.Probe{
+		Handler: v1.Handler{
+			Exec: &v1.ExecAction{
+				Command: []string{"/opt/nedge/sbin/grpc-proxy-liveness.sh"},
```

Comment:
ok, this change requires edgefs image >= 1.2.31. make sense to update all the examples & docs with newer version before we merge?